README.md

```diff
@@ -164,6 +165,15 @@ Notes:
 - If you run `rs train` in an environment without X11 you need to set `export MPLBACKEND="agg"` for charts, see [the matplotlib docs](https://matplotlib.org/faq/howto_faq.html#matplotlib-in-a-web-application-server).
 
 
+### rs export
+
+Exports a trained model in [ONNX](https://onnx.ai/) format for prediction across different backends (like Caffe2, TensorFlow).
+
+The result of `rs export` is an ONNX GraphProto `.pb` file which can be used with the ONNX ecosystem.
+
+Note: the `rs predict` tool works with `.pth` checkpoints. In contrast to these `.pth` checkpoints the ONNX models neither depent on PyTorch or the Python code for the model class and can be used e.g. in resource constrained environments like AWS Lambda.
```

Comment:
Should we move the `rs predict` tool to work with ONNX model instead of the current `.pth` checkpoints?

Reply:
There is currently a problem with the PyTorch <-> onnx native libraries which won't allow us to import both at the same time. That's quite unfortunate and tracked in: https://github.com/pytorch/pytorch/issues/8585.

In addition we need to do extensive tests and benchmarks before permanently switching to the onnx format for model storage in our pipeline.

The main reason for this pull request is having a quick tool to export to onnx, where we then can test and prototype solutions only based on the exported onnx file without any other dependencies.